packages/atomic/cypress/integration/result-list/result-table.cypress.ts

```diff
@@ -0,0 +1,102 @@
+import {generateComponentHTML, TestFixture} from '../../fixtures/test-fixture';
+import {getDeepText} from '../../utils/elementUtils';
+import * as CommonAssertions from '../common-assertions';
+import {resultTextComponent} from './result-components/result-text-selectors';
+import {ResultListSelectors} from './result-list-selectors';
+import {addResultTable} from './result-table-actions';
+import {ResultTableSelectors} from './result-table-selectors';
+
+describe('Result List (Table) Component', () => {
+  it('should load', () => {
```

Comment:
Thinking about these "should load" tests, all other tests kind of depend on the element loading. Do we need to have a unique setup for this, which amount to more test time, or can we put the expectation in another "describe" block?